WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt

```diff
@@ -915,6 +878,101 @@ secret_taoism_community = {
 	}
 }
 
+<<<<<<< HEAD
```

Comment:
Here is my 5-day old post initially notifying you of this, which has stayed throughout the pull-requests: https://github.com/loup99/WtWSMS/pull/224#discussion_r126287245